### PR TITLE
Fix the broken links in overview.md and Fixing typo error in Calibrator.py

### LINKED ISSUE #58383: Misspelling of the word "argument"
https://github.com/tensorflow/tensorflow/blob/cdf72454805f790b97ad731f5ea9a400ecbbc26d/tensorflow/lite/python/optimize/calibrator.py#L97

### PR DESCRIPTION
**Pull request by tiruk007**
Fixed the broken links for TensorFlow Lite Task Library in line 55 , TensorFlow Lite Interpreter Java API in line 58 and TensorFlow Lite Interpreter Swift API in line 72 and 

Fixing typo error `arugment` to `argument` in calibrator.py. Fixes #58383

### CODE REVIEW

**Review by sirakiin — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @tiruk007 Can you please resolve conflicts? Thank you!

**Comment by tiruk007** [comment]
Hi @gbaned 
Resolved the conflicts. Can you please check ?
Thanks !

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

File: tensorflow/lite/g3doc/examples/object_detection/overview.md [hunk 1]
```diff
@@ -52,10 +52,10 @@ started.
 #### Android
 
 You can leverage the out-of-box API from
-[TensorFlow Lite Task Library](../../inference_with_metadata/task_library/object_detector)
+[TensorFlow Lite Task Library](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/inference_with_metadata/task_library/object_detector.md)
```
Comment (sirakiin) [inline tensorflow/lite/g3doc/examples/object_detection/overview.md:55]: Can you keep the relative path?

File: tensorflow/lite/g3doc/examples/object_detection/overview.md [hunk 2]
```diff
@@ -52,10 +52,10 @@ started.
...
 to integrate object detection models in just a few lines of code. You can also
 build your own custom inference pipeline using the
-[TensorFlow Lite Interpreter Java API](../../guide/inference#load_and_run_a_model_in_java).
+[TensorFlow Lite Interpreter Java API](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/guide/inference.md#load-and-run-a-model-in-java).
```
Comment (sirakiin) [inline tensorflow/lite/g3doc/examples/object_detection/overview.md:58]: Same here for relative path.

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @sirakiin Can you please review this PR ? Thank you!

**Comment by pkgoogle** [comment]
This is superceded by https://github.com/tensorflow/tensorflow/commit/26612c8623c828c786dcebaa3151c5ea6d9ae0ec, closing.

**Pull request closed without merging**